src/data/ohlc_hourly/processing/main_process.py

```diff
@@ -1,7 +1,6 @@
 from data.storage.fetch_data_from_db import get_all_data_from_bigquery
 from data.storage.store_in_db import stream_data_to_bigquery, client
-from data.processing.process_fear_greed_data import ensure_bigquery_fear_greed_table
-from data.processing.process_ohlc_data import ensure_bigquery_ohlc_table
+from data.ohlc_hourly.processing.process_ohlc_data import ensure_bigquery_ohlc_table
```

Comment:
table id should be: **`raw_hourly_ohlc_data`**
 down

Reply:
ok